### PR TITLE
Move content from sklearn readme into sphynx project

### PR DESCRIPTION
**Pull request by eslesar-aws**
*Issue #, if available:*

*Description of changes:*
Migrate using content from /sklearn/README.rst into sphynx project

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [X] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,7 @@ CHANGELOG
 * doc-fix: move overview content in main README into sphynx project
 * bug-fix: pass accelerator_type in ``deploy`` for REST API TFS ``Model``
 * doc-fix: move content from tf/README.rst into sphynx project
+* doc-fix: move content from slearn/README.rst into sphynx project
```
Comment (nadiaya) [inline CHANGELOG.rst]: slearn -> sklearn

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=b410ea11-53fd-4438-b57a-a084cb6cc390%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cfa56bad-a0bb-46fb-9e98-8c85063d8125%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=26e8c85f-3ec4-4add-b598-460458f1f4ee%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**